handler.go

```diff
@@ -0,0 +1,34 @@
+package main
+
+import (
+	"fmt"
+	"net/http"
+	"context"
+
+	"github.com/jackc/pgx/v4"
+)
+
+
+
+func Home(w http.ResponseWriter, r *http.Request){
+	fmt.Println("hit home page")
```

Comment:
You probably just haven't gotten around to it, but you should probably use a logger instead of fmt.Println.